department-of-veterans-affairs/caseflow
# Update diagnostic codes from bgs

Resolves users not being able to see diagnostic codes in dropdown

### Description
Updated diagnostic codes. Queried all codes from BGS while we wait to dynamically update this list in #14870, #14871, #14872 & #14877

https://github.com/department-of-veterans-affairs/caseflow/issues/14877#issuecomment-712357013

### Acceptance Criteria
- [ ] DIAGNOSTIC_CODE_DESCRIPTIONS contains new diagnostic codes from BGS

### Testing Plan
1. Sign in as an attorney and go to any legacy case case with > 0 issues
1. Select decision ready for review and go to the select dispositions page
1. Click Edit issue on any issue
1. Search for diagnostic codes 7717 to 7724 in the dropdown to confirm they exist

### User Facing Changes

BEFORE|AFTER
 ---|---
![Screen Shot 2020-10-19 at 2 29 30 PM](https://user-images.githubusercontent.com/45575454/96496766-a8c59180-1217-11eb-8fa6-3c1f2d2d596b.png)|![Screen Shot 2020-10-19 at 2 27 50 PM](https://user-images.githubusercontent.com/45575454/96496768-a95e2800-1217-11eb-8baf-af5a089b0818.png)
